site/en/guide/create_op.md

```diff
@@ -47,7 +47,7 @@ To incorporate your custom op you'll need to:
     test the op in C++. If you define gradients, you can verify them with the
     Python `tf.test.compute_gradient_error`.
     See
-    [`relu_op_test.py`](https://www.tensorflow.org/code/tensorflow/python/kernel_tests/relu_op_test.py) as
+    [`relu_op_test.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/nn_ops/relu_op_test.py) as
```

Comment:
Thanks for spotting this.

@markmcd:

- The following link doesn't work: https://www.tensorflow.org/code/tensorflow/python/kernel_tests/relu_op_test.py -> redirects to https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/kernel_tests/relu_op_test.py
- Should a GitHub-hosted link be OK for now? Also see the first result in https://www.tensorflow.org/s/results?q=relu_op_test.py

Reply:
Yeah it's fine to use GitHub links directly. Looks like the /code/ redirect was there to support a very old version of the site.

Reply:
Thanks @tiruk007 @markmcd LGTM